### DISCUSSION TITLE
Hoppscotch.io

### DISCUSSION THREAD

**Discussion by justTOBBI** [catppuccin/catppuccin#1907] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

via Stylus Userthemes

### Has the requested port been themed already?

https://github.com/justTOBBI/hoppscotch

### Any additional comments?

Have a beautiful day, my friends ☕

**Comment by sgoudham** [comment]
Transferred to #1908
